eureka-client/src/main/java/com/netflix/discovery/DiscoveryClient.java

```diff
@@ -196,6 +196,30 @@
 
         @Inject(optional = true)
         private Provider<HealthCheckHandler> healthCheckHandlerProvider;
+
+        public EventBus getEventBus() {
```

Comment:
They aren't an "alternative to DiscoveryClientOptionalArgs" as they are methods on the class.  I'll add the new constructor instead.